txbits/test/WalletSpec.scala

```diff
@@ -29,9 +29,9 @@ import play.api.test._
 
 import org.specs2.mock._
 import wallet.{ WalletModel, Wallet }
-import com.googlecode.jsonrpc4j.JsonRpcHttpClient
-import com.fasterxml.jackson.databind.node.ObjectNode
-import com.fasterxml.jackson.databind.ObjectMapper
+import _root_.com.googlecode.jsonrpc4j.JsonRpcHttpClient
```

Comment:
It compiled fine before but IntelliJ gets the namespaces confused and tries to get it from `test.com`, which results in `Cannot resolve symbol` errors in the IDE.